pkgs/development/python-modules/construct/2.10.54.nix

```
@@ -14,9 +14,20 @@ buildPythonPackage rec {
     sha256 = "1mqspsn6bf3ibvih1zna2glkg8iw7vy5zg9gzg0d1m8zcndk2c48";
   };
 
-  checkInputs = [ pytestCheckHook pytest-benchmark enum34 numpy arrow ruamel-yaml ];
+  checkInputs = [ pytestCheckHook pytest-benchmark enum34 numpy ];
 
-  disabledTests = lib.optionals stdenv.isDarwin [ "test_multiprocessing" ];
+  # these have dependencies that are broken on Python 2
+  disabledTestPaths = [
+    "tests/gallery/test_gallery.py"
+    "tests/test_benchmarks.py"
+    "tests/test_compiler.py"
+  ];
+
+  disabledTests = [
+    "test_timestamp"
+  ] ++ (lib.optionals stdenv.isDarwin [
+    "test_multiprocessing"
+  ]);
```

```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    "test_multiprocessing"
  ];
```
nit: Those are optional.

---

True. I have to remember they're only needed for `with`.